WebRoot/admin/companies/view.php

```diff
@@ -14,10 +14,8 @@
 			<?php include 'Templates/navbar admin.php' ?>
 		</nav>
 		<main>
-			<div class="information-error-box" style="display:none" id="MsgBox">
-				<div class="information-error-vertical"></div>
-				<h2 id="MsgTxt"></h2>
-			</div>
```

Comment:
This is used by the JavaScript scraper to display error messages if something happen. It's why the style is initially set to "display:none"
See `WebRoot/static/dhtml/admin.js`

Reply:
I think it like to show on `admin/view.php` and for it to be visible.

Reply:
Yes, it is exactly the same system but on this page it's to display an error message if something happens when you add a new company

Reply:
I could re-add it to the `company view.php`

Reply:
I put a placeholder for now. I think i have a nice way to display it in the future. When we can add companies, i will make it better. I think it is good for now.